File: JAVA/Program04.java
////////////////////////////////////////////////////////////////////
//                                                                //
//  File name :     Program04.java                                //
//  Description :   Used to Display Factors of Given Number       //
//  Author :        Pallavi Omprakash Malewar                     //
//  Date :          24/08/2025                                    //
//                                                                //
////////////////////////////////////////////////////////////////////


import java.util.Scanner;

class Number
{
    public void displayFactors(int iNo)
    {
        int iCount = 0;

        for(iCount = 1 ; iCount <= (iNo / 2) ; iCount++)
        {
            if((iNo % iCount) == 0)
            {
                System.out.print(iCount + "\t");
            }
        }
        System.out.println(iNo);        // To Print the number itself as it is a factor

    }
}

class Program04
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.print("Enter the Number : ");
        iValue = sobj.nextInt();

        Number nobj = new Number();

        nobj.displayFactors(iValue);

        sobj.close();
    }
}

File: CPP/Program04.cpp
///////////////////////////////////////////////////////////
//                                                       //
//  File name :     Program04.cpp                        //
//  Description :   Used to do find largest number       //
//  Author :        Pallavi Omprakash Malewar            //
//  Date :          05/09/2025                           //
//                                                       //
///////////////////////////////////////////////////////////


#include<iostream>
using namespace std;

int Largest(int iNo1 , int iNo2 , int iNo3)
{
    if((iNo1 > iNo2) && (iNo1 > iNo3))
    {
        return iNo1;
    }
    else if((iNo2 > iNo3) && (iNo2 > iNo1))
    {
        return iNo2;
    }
    else
    {
        return iNo3;
    }
}

int main()
{

    cout<<Largest(14, 18, 11)<<endl;
    cout<<Largest(45, 23, 78)<<endl;

    return 0;
}
